### ISSUE TITLE
Adding users reviews page

### ISSUE THREAD

**Issue by AakankshaKamal** [Namanl2001/MERN-Gurujii-dev#134]
**Describe the feature you'd like to add**
At the home page,at the navbar, we can provide a link that will take the user to a page that will contain the user's reviews regarding the website including teachers and students, and how it has helped them. It will be quite attractive for users using it for the first time.

**Comment by guptaji6489** [comment]
That's nice

**Comment by sagar-barapatre** [comment]
Shouldn't that thing be placed in the home page only. #JustAsking @AakankshaKamal

**Comment by guptaji6489** [comment]
> Shouldn't that thing be placed in the home page only. #JustAsking @AakankshaKamal

Ya, Not in the Navbar

**Comment by AakankshaKamal** [comment]
We can do that also .  I suggested a new page for it  just that it would contain users reviews  so the home page does not look crowded.

**Comment by sagar-barapatre** [comment]
No I was thinking similar to the reviews shown by courseera on their website. That looks elegant and simple.
![Screenshot 2021-04-09 at 9 08 15 AM](https://user-images.githubusercontent.com/64516304/114124988-2dfe4900-9913-11eb-952f-f04fcb20c7cc.png)
What say people ?

**Comment by AakankshaKamal** [comment]
This looks good

**Comment by guptaji6489** [comment]
> No I was thinking similar to the reviews shown by courseera on their website. That looks elegant and simple.
> ![Screenshot 2021-04-09 at 9 08 15 AM](https://user-images.githubusercontent.com/64516304/114124988-2dfe4900-9913-11eb-952f-f04fcb20c7cc.png)
> What say people ?

Nice

**Comment by guptaji6489** [comment]
Want to work on this @AakankshaKamal

**Comment by AakankshaKamal** [comment]
Yes I would like to work on it

**Comment by sagar-barapatre** [comment]
Okay thank you ! You can start working.

**Comment by sagar-barapatre** [comment]
@AakankshaKamal madam what's the status ? How work is done ?

**Comment by AakankshaKamal** [comment]
Yes Sir I am working on it , need to connect it to  backend so I was stuck in that.I'll need 2 days approx for completing it.

**Comment by sagar-barapatre** [comment]
@AakankshaKamal is all okay ? Any help ??

**Comment by AakankshaKamal** [comment]
I could not work on this issue right now.Please unassign me

**Comment by Manoramsharma** [comment]
@Namanl2001 @sagar-barapatre want to contribute , is review page UI already with us ?

**Comment by Namanl2001** [comment]
@Manoramsharma nope it's not with us. You are welcome to contribute 🤗. Thanks

**Comment by Manoramsharma** [comment]
@Namanl2001 want to contribute
